### PR TITLE
Update CLI.md

### PR DESCRIPTION
**Pull request by hosseinsia**
Update the suggested command template for "trust keys", to make it consistent with other examples.

Please fill in the fields below to submit a pull request.  The more information
that is provided, the better.

Fixes #<ISSUE NUMBER>

**Description of the changes being introduced by the pull request**:

**Please verify and check that the pull request fulfills the following
requirements**:

- [ ] The code follows the [Code Style Guidelines](https://github.com/secure-systems-lab/code-style-guidelines#code-style-guidelines)
- [ ] Tests have been added for the bug fix or new feature
- [ ] Docs have been added for the bug fix or new feature

### CODE REVIEW

File: docs/CLI.md
Comment (MVrachev) [inline docs/CLI.md]:
```suggestion
$ repo.py --trust --pubkeys </path/to/foo_key.pub> --role <rolename>
```
Thanks for your pr @hosseinsia!
I am not sure what do you express with the `+`?

Comment (hosseinsia) [reply docs/CLI.md]: Hey @MVrachev, by + i meant it can be more than one but I guess people also get the point without that.

Comment (MVrachev) [reply docs/CLI.md]: I see, I don't mind the `+`.
I am not a maintainer, so we would have to wait for @jku, @joshuagl, or somebody else from the maintainers to approve.

**Review by trishankatdatadog — APPROVED** [review]

**Comment by trishankatdatadog** [comment]
@hosseinsia can you fix DCO? You just need to `--sign-off` your commits. Please see the DCO bot message 🙂

**Review by joshuagl — APPROVED** [review]
Thanks for the contribution @hosseinsia !

**Pull request merged**
